bot/exts/filters/filtering.py

```diff
@@ -157,6 +158,17 @@ def mod_log(self) -> ModLog:
         """Get currently loaded ModLog cog instance."""
         return self.bot.get_cog("ModLog")
 
+    @Cog.listener()
+    async def on_member_join(self, member: Member) -> None:
+        """Check if the joining member has a bad word in their name."""
+        await self.check_bad_words_in_name(member)
+
+    @Cog.listener()
+    async def on_member_update(self, before: Member, after: Member) -> None:
+        """If the member has updated their name, check if it has a bad word in it."""
+        if before.display_name != after.display_name:
```

Comment:
When the member isn't in cache `before is after` will be True afaik. Do we want to run the check if there's a possiblity that the name or nickname was changed? This isn't very expensive right?

Reply:
following a discussion in a mod channel, we decided we don't want to alert on member join/edit, as those members may never have even spoken in our community, so it's not a burden we want to put on the mod team.